cluster/examples/workloads/kubernetes/wordpress/aws/provider.yaml

```diff
@@ -33,6 +33,7 @@ parameters:
 provisioner: rdsinstance.database.aws.crossplane.io/v1alpha1
 providerRef:
   name: aws-provider
+  namespace: crossplane-system
 reclaimPolicy: Delete
 ---
 apiVersion: core.crossplane.io/v1alpha1
```

Comment:
Worth noting that this will need to change with the implementation of strongly typed resource classes. I would be fine with merging this without that update due to the fact that it doesn't "work less" than the current state of the example (which does not currently work on `master`). Alternatively, I could sync with you and we could move to strongly typed resource classes for just this example and the rest would follow with my forthcoming PR. We will also need to go back to not sharing this `provider.yaml` between both aws and gcp because they will use different resource classes. Thank you for your diligent effort here while the repo has been in a state of transition :)

Reply:
Another option would be to just implement the changes you specified in (2) and (4) in the PR body above and we could likely go ahead and get this merged!

Reply:
I'd vote to do all the strongly type related changes in another pr, as it would confuse the purpose of this pr.

regarding the `provider.yaml` file location, it's not currently being shared. Each `gcp` and `aws` each have their own copy

Reply:
Cool! And yes you are right about `provider.yaml`, I had misinterpreted which files were being moved